tests/Controller/CountryApiTest.php

```diff
@@ -11,7 +11,6 @@
 
 namespace Sylius\Tests\Controller;
 
-use Lakion\ApiTestCase\JsonApiTestCase;
```

Comment:
I removed the duplication of authorization token and headers to `Sylius\Tests\Controller\JsonApiTestCase` which extends from `\Lakion\ApiTestCase\JsonApiTestCase`